services/task/task.py

```diff
@@ -134,6 +134,7 @@ def upload_signature(
         commitid,
         report_type=None,
         report_code=None,
+        arguments=None,
```

Comment:
Do we use these in Worker?

Reply:
Not yet, but I would like to make the task workflow simpler and make the batching more explicit.

https://github.com/codecov/engineering-team/issues/2618 explains some of my ideas I am working towards.

In particular, I would love to get rid of the `Upload -> N * UploadProcessor -> UploadFinisher` fanout / map/reduce to a different strategy. Passing the `arguments` explicitly is a first step towards that.